README.md

```diff
@@ -22,65 +86,113 @@ For the latter you will need:
+
+```yaml
+service_type: gitlab
+gitlab:
+  api_token: <your-api-key>
 ```
 
-This command is also included in every `make run`, so when you re-run the app using the makefile, these values are updated, too.
-They are exposed in the root endpoint mentioned above.
+This way, you're saying that the connector is for GitLab, and there you're providing the API key for GitLab. And it also means that the `GitLab::Connector` class will have a `@local_configuration` variable that will contain whatever is in the `gitlab` section of the configuration file. So inside the connector class, you can access the API key like this:
 
-### Configuration
+```ruby
+@local_configuration[:api_token]
+```
 
-By design, we try to avoid duplicating any metadata in the project, like its
-**version**. For this reason, we have one single configuration file in
-`config/connectors.yml` that contains all the metadata needed for runtime and
-buildtime. It can be used from the dev tree or in a production deployment.
+### Testing the integration
 
-The Gem spec and the connectors use that config file to get
-the metadata they need.
+If the connector synchronizes successfully, the documents should appear in `Enterprise Search
+Content - Elasticsearch indices - your-index-name` section, on the `Documents` tab.
 
-The build process might change it on-the-fly when the Gem is created but will
-not change the one in the dev tree.
+If the connector didn't specify any configurable fields, it will go into the `connected` status and will try to sync, without requiring any additional configuration via the Kibana UI.
 
-When the connector application is launched, it will pick `config/connectors.yml` by default,
-but you can provide your own configuration file by using the **CONNECTORS_CONFIG** env.
+If the connector fails to synchronize, it will change its status to `error`.
 
+## Moving to production
 
-### Connector protocol
+After you've implemented everything correctly, you're ready to go to Production. Before you do this, make sure the following topics are covered:
 
-These connectors are built according to our [Connector Protocol](./docs/CONNECTOR_PROTOCOL.md). If you want to build your own connector or update an existing, please make sure to adhere to that protocol.
+- No sensitive values are committed to source code repositories.
+- No unsecured values are exposed via configurable fields.
+- The connector is properly authenticated to the third-party data source.
+- The connector implements the connector protocol correctly and does appropriate error handling.
 
 ### Contribute 🚀
 We welcome contributors to the project. Before you begin, please read the [Connectors Contributor's Guide](./docs/CONTRIBUTING.md).
```

Comment:
Please update the support.md file to link to https://discuss.elastic.co/c/enterprise-search/84, instead of Workplace Search and to the enterprise-search slack channel.